components/script/dom/event.rs

```diff
@@ -126,6 +126,10 @@ impl Event {
         self.cancelable.set(cancelable);
     }
 
+    pub fn set_target(&self, target_: &EventTarget) {
```

Comment:
This is redundant change, will remove it.

Reply:
https://github.com/servo/servo/pull/34524/commits/7590972396652642f182591d2b3d470e3ad8b469